src/db.ls

```diff
@@ -67,16 +67,58 @@
     db.DB = {}
     minimatch = require \minimatch
     try
-      db.DB = JSON.parse do
-        require \fs .readFileSync "#dataDir/dump.json" \utf8
-      console.log "==> Restored previous session from JSON file"
+      db.DB =
+        save_timestamps: {}
+        timestamps: {}
+      if fs.existsSync "#dataDir/dump/"
+        fs.readdirSync("#dataDir/dump/").forEach (f) ->
+          key = f.split(".")[0]
+          type = key.split("-")[0]
+          id = key.split("-")[1]
+          db.DB.timestamps["timestamp-#id"] = 0
+          db.DB.save_timestamps["timestamp-#id"] = 0
+          if type is "snapshot"
+            db.DB[key] = fs.readFileSync("#dataDir/dump/#key.txt", \utf8)
+          else if type is "audit"
+            db.DB[key] = fs.readFileSync("#dataDir/dump/#key.txt", \utf8).split("\n")
```

Comment:
The `split` produces a blank string in the resulting array:

```json
[ "set A1 value n 1",
  "set A2 value n 2",
  "set A3 value n 3",
  "" ]
```

One could filter like so:

```ls
v =
    switch type
    | \snapshot =>
        fs.read-file-sync "#dataDir/dump/#key.txt", \utf8
    | \audit =>
        fs.read-file-sync "#dataDir/dump/#key.txt", \utf8
            .split "\n"
            .filter (.length) # remove any blanks
    | _ => void
db.DB[key] = v if v?
```